.github/workflows/php.yml

```diff
@@ -42,25 +44,12 @@ jobs:
       with:
         php-version: ${{ matrix.php }}
 
-    - name: Validate composer.json and composer.lock
-      run: composer validate
-      
-    - name: Cache Composer packages
-      id: composer-cache
-      uses: actions/cache@v3
-      with:
-        path: vendor
-        key: ${{ runner.os }}-php${{ matrix.php }}-${{ hashFiles('**/composer.lock') }}
-        restore-keys: |
-          ${{ runner.os }}-php${{ matrix.php }}-
-
     - name: Install node dependencies
       if: steps.node-cache.outputs.cache-hit != 'true'
       run: npm install
 
     - name: Install composer dependencies
-      if: steps.composer-cache.outputs.cache-hit != 'true'
-      run: composer update --prefer-dist --no-progress -n
```

Comment:
I think we should stick to composer update? We can just clear the if condition now that we know for sure the lock file won't be available.

Reply:
Let me test a bit on diff repo, i have same though with you that it should be fine if we still use the same way with before. Just want to make sure if it hit the cache or not

Reply:
in case of having no `composer.lock`, `composer install` will behave similar with `composer update`

Reply:
But now that we've restored the composer.lock?

Reply:
i restored the package-lock.json as we just use it for running linter and test.
much easier to setup the cache for node_modules